browser_history/browsers.py

```diff
@@ -121,3 +121,27 @@ class Edge(Browser):
 
     history_file = Chrome.history_file
     history_SQL = Chrome.history_SQL
+
+class Opera(Browser):
+    """Opera Browser
+
+    Supported platforms (TODO: Mac OS support)
+
+    * Linux, Windows
+
+    Profile support: No
+    """
+    name = "Opera"
+
+    linux_path = '.config/opera'
+    windows_path = 'AppData/Roaming/Opera Software/Opera Stable'
+    
+    profile_support = False
+    
+    history_file = 'History'
+    history_SQL = """SELECT
+            datetime(visits.visit_time/1000000-11644473600, 'unixepoch', 'localtime') as 'visit_time',
+        urls.url from urls,visits
+        WHERE urls.id = visits.url ORDER BY visit_time DESC"""
```

Comment:
Since this is the exact same as `Chrome`, you could just do the same as Chromium or Edge.
```suggestion
    history_SQL = Chrome.history_SQL
```

Reply:
Yeah makes sense

Reply:
Changed